<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel { font-size: 14pt; }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="device_info">
     <property name="text">
      <string>current-device %1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="manageDevices">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
	border: 0.05em solid #CA0B5E;
	border-radius: 0.25em;
	color: #CA0B5E;
	padding: 0.5em;
	font-size: 13pt;
	outline: none;
}
QWidget:focus {
	border: 0.1em solid #CA0B5E;
}
QWidget:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
</string>
       </property>
       <property name="text">
        <string>manage-devices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deviceIdInput">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>34</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 0.05em solid black;
	border-radius: 0.25em;
	padding: 0.25em;
	font-size: 14pt;
	outline: none;
}
QLineEdit:focus {
	border: 0.05em solid #CA0B5E;
}
QLineEdit[error=true] {
	color: black;
	background-color: rgb(255, 230, 230);
}</string>
       </property>
       <property name="placeholderText">
        <string>device-id-placeholder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deviceOk">
       <property name="styleSheet">
        <string notr="true">QWidget {
	border: 0.05em solid #CA0B5E;
	border-radius: 0.25em;
	color: #CA0B5E;
	padding: 0.5em;
	font-size: 13pt;
	outline: none;
}
QWidget:focus {
	border: 0.1em solid #CA0B5E;
}
QWidget:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deviceCancel">
       <property name="styleSheet">
        <string notr="true">QWidget {
	border: 0.05em solid #CA0B5E;
	border-radius: 0.25em;
	color: #CA0B5E;
	padding: 0.5em;
	font-size: 13pt;
	outline: none;
}
QWidget:focus {
	border: 0.1em solid #CA0B5E;
}
QWidget:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>manageDevices</sender>
   <signal>clicked()</signal>
   <receiver>DeviceWidget</receiver>
   <slot>manageDevices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deviceOk</sender>
   <signal>clicked()</signal>
   <receiver>DeviceWidget</receiver>
   <slot>applyDeviceUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deviceCancel</sender>
   <signal>clicked()</signal>
   <receiver>DeviceWidget</receiver>
   <slot>cancelDeviceUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>manageDevices()</slot>
  <slot>applyDeviceUpdate()</slot>
  <slot>cancelDeviceUpdate()</slot>
 </slots>
</ui>
